x-pack/plugins/apm/kibana.json

```diff
@@ -5,6 +5,6 @@
   "kibanaVersion": "kibana",
   "configPath": ["xpack", "apm"],
   "ui": false,
-  "requiredPlugins": ["apm_oss", "data", "home", "licensing"],
+  "requiredPlugins": ["apmOss", "data", "home", "licensing"],
   "optionalPlugins": ["cloud", "usageCollection"]
 }
```

Comment:
I think the build is failing because the following to instances of `apm_oss` also need to be changed:

https://github.com/elastic/kibana/blob/7bafeb1d6f562a7e6b3c893ad7fc51b0a2e5de6a/x-pack/plugins/apm/server/plugin.ts#L45

https://github.com/elastic/kibana/blob/7bafeb1d6f562a7e6b3c893ad7fc51b0a2e5de6a/x-pack/plugins/apm/server/plugin.ts#L54

@joshdover do you agree?

Reply:
@sqren I think you're right. This is happening on startup for me:

```
server    log   [08:08:06.407] [fatal][root] TypeError: Cannot read property 'config$' of undefined
    at APMPlugin.setup (/Users/smith/Code/kibana/x-pack/plugins/apm/server/plugin.ts:54:57)
    at PluginWrapper.setup (/Users/smith/Code/kibana/src/core/server/plugins/plugin.ts:100:26)
    at PluginsSystem.setupPlugins (/Users/smith/Code/kibana/src/core/server/plugins/plugins_system.ts:91:25)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
Oh! I can do that.

Reply:
Yes that sounds right